deployment/kuadrant-openshift/install-prometheus-grafana.sh

```
@@ -0,0 +1,414 @@
+  log_info "🔍 Checking OpenShift user workload monitoring configuration..."
+
+  # Check if user workload monitoring is enabled
+  if $KCMD get configmap cluster-monitoring-config -n openshift-monitoring -o yaml 2>/dev/null | grep -q "enableUserWorkload: true"; then
+    log_success "✅ OpenShift user workload monitoring is properly configured"
+    return 0
+  fi
+
+  log_info "⚠️ OpenShift user workload monitoring is not enabled"
+  log_info "ℹ️ Enabling user workload monitoring allows metrics collection for the MAAS chart."
+
+  local monitoring_yaml=$(cat <<EOF
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)